src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs

```diff
@@ -690,6 +709,63 @@ public override BoundNode VisitBaseReference(BoundBaseReference node)
                 : FramePointer(node.Syntax, _topLevelMethod.ContainingType); // technically, not the correct static type
         }
 
+        public BoundStatement RewriteLocalFunctionDefinitions(BoundStatement loweredBody)
```

Comment:
Minor nitpick: should this be called `RewriteLocalFunctionDefinitions` or `RewriteLocalFunctionReferences`? Definitions makes it sound like it's rewriting the declarations of local functions, which by the looks of it, this rewrites references.